integration_tests/src/main/python/hybrid_parquet_test.py

```diff
@@ -144,7 +140,34 @@ def test_hybrid_parquet_read_fallback_to_gpu(spark_tmp_path, parquet_gens):
         non_exist_classes='HybridFileSourceScanExec',
         conf={
             'spark.sql.sources.useV1SourceList': 'parquet',
-            'spark.rapids.sql.parquet.useHybridReader': 'true',
+            'spark.rapids.sql.hybrid.parquet.enableReader': 'true',
+        })
+
+
+# Test the preloading feature with extreme tiny target batch size (and source batch size), creating
```

Comment:
In the near future, should we reuse [Parquet test](https://github.com/NVIDIA/spark-rapids/blob/branch-25.04/integration_tests/src/main/python/parquet_test.py) instead of a newly introduced [hybrid_parquet_test.py](https://github.com/NVIDIA/spark-rapids/pull/11998/files#diff-d79c26eac1b50d8683a54900b348f34921603bfeb042126f44881430f2cd0b25)?

Reply:
It doesn't seem to be doable since HybridScan only supports a subset of datatypes which supported GpuScan